pkg/listener/mysql.go

```diff
@@ -780,8 +781,22 @@ func (l *MysqlListener) ExecuteCommand(ctx context.Context, c *mysql.Conn, data
 			l.stmts.Delete(stmtID)
 		}
 	case constant.ComStmtSendLongData: // no response
-		values := make([]byte, len(data))
-		copy(values, data)
+		if len(data) < 6 {
```

Comment:
```suggestion
		if len(data) < minPKLen {
```

Reply:
long data packet length must be greater than 6, some others can be less than 6.